README.md

```diff
@@ -41,7 +43,9 @@ or
 @import SplunkOtel;
 ...
 // Your beaconUrl and rumAuth will be provided by your friendly Splunk representative
-[SplunkRum initializeWithBeaconUrl: @"https://rum-ingest.us0.signalfx.com/v1/rum" rumAuth: @"ABCD..." options: nil];
+SplunkRumBuilder *builder = [SplunkRumBuilder withBeaconUrl: @"https://rum-ingest.us0.signalfx.com/v1/rum" rumAuth: @"ABCD..."];
+// Call functions to configure additional options
+[builder build];
```

Comment:
What would the ObjC equivalent of setting a global attribute in Swift?

Our previous code was:

```objc
@import SplunkOtel;

//...
SplunkRumOptions *options = [[SplunkRumOptions alloc] init];
options.environment = @"<environment-name>";
[SplunkRum initializeWithBeaconUrl:@"https://rum-ingest.<realm>.signalfx.com/v1/rum" rumAuth: @"<rum-token>" options: options];
```

Reply:
```objc
@import SplunkOtel;

SplunkRumBuilder *builder = [SplunkRumBuilder withBeaconUrl: @"https://rum-ingest.us0.signalfx.com/v1/rum" rumAuth: @"ABCD..."];
[builder withGlobalAttributes: [:]]
[builder build];
```

Reply:
Would you still do `options.environment = @"<environment-name>";`?